spring-integration-core/src/main/java/org/springframework/integration/aggregator/AbstractCorrelatingMessageHandler.java

```diff
@@ -842,6 +844,10 @@ protected void completeGroup(Object correlationKey, MessageGroup group, Lock loc
 				}
 				result = messageBuilder.popSequenceDetails();
 			}
+			else if (this.popSequence && partialSequence == null && result instanceof Message && (ObjectUtils.nullSafeEquals(((Message) result).getHeaders().get(IntegrationMessageHeaderAccessor.SEQUENCE_DETAILS), message.getHeaders().get(IntegrationMessageHeaderAccessor.SEQUENCE_DETAILS)))) {
```

Comment:
This line's too long (> 120).

Also, the if/elses can be restructured to avoid double checks on `this.popSequence && partialSequence == null && result instanceof Message`

Reply:
I will restructure the If else, I kept it like this to ensure minimum changes for ease of review.